gremlin-python/src/main/python/gremlin_python/driver/serializer.py

```diff
@@ -72,10 +75,10 @@ def close(self, args):
     def bytecode(self, args):
         gremlin = args['gremlin']
         args['gremlin'] = self._writer.to_dict(gremlin)
-        aliases = args.get('aliases', '')
-        if not aliases:
-            aliases = {'g': 'g'}
-        args['aliases'] = aliases
+        bindings = args.get('bindings', '')
+        if not bindings:
+            bindings = {'g': 'g'}
```

Comment:
There's a "g" in the request now that is separate from bindings. The new "g" takes precedence over anything that was set in bindings (overwrites any 'g' key in bindings).

Reply:
Ah, this was a left over change from me playing with the code at the beginning. This is part of the session op processor code that will be removed.